marlowe-runtime/tx/Language/Marlowe/Runtime/Transaction/Server.hs

```diff
@@ -105,9 +105,7 @@ import Language.Marlowe.Runtime.Transaction.Query
   (LoadMarloweContext, LoadWalletContext, lookupMarloweScriptUtxo, lookupPayoutScriptUtxo)
```

Comment:
Changes in this file:

- Move server sources to component outputs
- Connection refactor